dmsg-http_test.go

```diff
@@ -19,7 +20,7 @@ const (
 	testPort uint16 = 8081
 )
 
-func TestDMSGClient(t *testing.T) {
+func TestDmsgHTTP(t *testing.T) {
```

Comment:
Guess this should be either TestDmsgHttp or TestHTTPOverDMSG / TestHTTPUsingDMSG

Reply:
I looked at the naming in the standard library and it was HTTP in that